metriq_gym/job_manager.py

```diff
@@ -64,12 +67,17 @@ def _load_jobs(self):
         self.jobs = []
         if os.path.exists(self.jobs_file):
             with open(self.jobs_file) as file:
-                for line in file:
+                for line_no, line in enumerate(file, start=1):
```

Comment:
Let's align with the 0-based numbering we use when listing jobs (e.g. https://github.com/unitaryfoundation/metriq-gym/blob/c051126cd78671801210dde2430491edee08c670/metriq_gym/cli.py#L60). We could convert both to 1-based (my preference), but the most important thing is that we are coherent.

Reply:
nit: rename `line_no` to `line_number`